llvm/test/Transforms/ObjCARC/rv.ll

```diff
@@ -103,15 +103,15 @@ entry:
 ; directly to a return value.
 
 ; TODO
-; COM: CHECK: define ptr @test5
-; COM: CHECK: call ptr @returner()
-; COM: CHECK-NEXT: ret ptr %call
-;define ptr @test5() {
-;entry:
-;  %call = call ptr @returner()
-;  %0 = call ptr @llvm.objc.retainAutoreleaseReturnValue(ptr %call) nounwind
-;  ret ptr %0
-;}
+; CHECK: define ptr @test5
+; CHECK: call ptr @returner()
+; CHECK-NEXT: ret ptr %call
+define ptr @test5() {
+entry:
+  %call = call ptr @returner()
+  %0 = call ptr @llvm.objc.retainAutoreleaseReturnValue(ptr %call) nounwind
+  ret ptr %0
+}
```

Comment:
fails because of of CHECK-NEXT:
```
90: define ptr @test5() { 
91: entry: 
92:  %call = call ptr @returner() 
93:  %0 = call ptr @llvm.objc.retainAutoreleaseReturnValue(ptr %call) #0 
94:  ret ptr %0 
```